src/build.c

```diff
@@ -1278,27 +1261,25 @@ static void set_file_error_from_errno(GError **error, gint err, const gchar *pre
 
 
 /* write a little shellscript to call the executable (similar to anjuta_launcher but "internal")
- * fname is the full file name (including path) for the script to create */
-static gboolean build_create_shellscript(const gchar *fname, const gchar *working_dir, const gchar *cmd, gboolean autoclose, GError **error)
+ * it returns the full file name (including path) of the created script in the locale encoding */
+static gchar *build_create_shellscript(const gchar *working_dir, const gchar *cmd, gboolean autoclose, GError **error)
```

Comment:
Now reading the comment again, I was actually talking about the return value. But I should mention the parameter encoding too.